RunningOrder/Story/Story.swift

```diff
@@ -9,7 +9,8 @@
 import Foundation
 
 struct Story {
-    var name: String
+    let sprintId: UUID
```

Comment:
just a string

Reply:
or to ensure this is ok, we could use `extension Sprint { typealias Id = String }` and use here Sprint.Id as type